docs/source/api_ref_models.rst

```diff
@@ -225,14 +226,42 @@ To download the Qwen2 1.5B model, for example:
 
     qwen2.qwen2
     qwen2.lora_qwen2
-    qwen2.qwen2_7b
     qwen2.qwen2_0_5b
-    qwen2.qwen2_1_5b
-    qwen2.lora_qwen2_7b
     qwen2.lora_qwen2_0_5b
+    qwen2.qwen2_1_5b
     qwen2.lora_qwen2_1_5b
+    qwen2.qwen2_7b
+    qwen2.lora_qwen2_7b
     qwen2.qwen2_tokenizer
 
+    |
+
+    qwen2_5.qwen2_5_0_5b
+    qwen2_5.lora_qwen2_5_0_5b
+    qwen2_5.qwen2_5_1_5b_base
+    qwen2_5.qwen2_5_1_5b_instruct
+    qwen2_5.lora_qwen2_5_1_5b_base
+    qwen2_5.lora_qwen2_5_1_5b_instruct
+    qwen2_5.qwen2_5_3b
+    qwen2_5.lora_qwen2_5_3b
+    qwen2_5.qwen2_5_7b_base
+    qwen2_5.qwen2_5_7b_instruct
+    qwen2_5.lora_qwen2_5_7b_base
+    qwen2_5.lora_qwen2_5_7b_instruct
+    qwen2_5.qwen2_5_14b_base
+    qwen2_5.qwen2_5_14b_instruct
+    qwen2_5.lora_qwen2_5_14b_base
+    qwen2_5.lora_qwen2_5_14b_instruct
+    qwen2_5.qwen2_5_32b_base
+    qwen2_5.qwen2_5_32b_instruct
+    qwen2_5.lora_qwen2_5_32b_base
+    qwen2_5.lora_qwen2_5_32b_instruct
+    qwen2_5.qwen2_5_72b_base
+    qwen2_5.qwen2_5_72b_instruct
+    qwen2_5.lora_qwen2_5_72b_base
+    qwen2_5.lora_qwen2_5_72b_instruct
+    qwen2_5.qwen2_5_tokenizer
```

Comment:
This is a bit hard on the eyes for me, I'd have to ctrl+f to navigate this.

Reply:
Is there also any reason not to split into separate Qwen 2, and Qwen 2.5 sections?

Reply:
llama3 and 3.1 are in one section, so I was following that. but yeah I think the feature gap is significant enough for two separate sections

Reply:
> This is a bit hard on the eyes for me, I'd have to ctrl+f to navigate this

not sure how else I can format it without just removing some builders

Reply:
I think I missed when we put 3 and 3.1 together but not exactly sure why we did that tbh. Imo it is better to keep things separate, especially given the large number of Qwen2.5 models. As for readability, we should probably ideally have more nested subsections for different model sizes? I think we kinda shot ourselves in the foot by using section header for our title.. we should refactor to make it a proper title in RST parlance (stolen from stackoverflow):

```
==============
Document title
==============
-----------------
Document subtitle
-----------------

Section
=======

Sub-section
-----------
```